packages/console/src/pages/Connectors/components/Guide/index.tsx

```diff
@@ -66,14 +69,26 @@ const Guide = ({ connector, onClose }: Props) => {
       return;
     }
 
-    const { id: connectorId } = connector;
+    return result;
+  };
+
+  const onSubmit = handleSubmit(async (data) => {
+    if (isSubmitting) {
+      return;
+    }
+
+    const { formItems, isStandard, id: connectorId } = connector;
+    const config = formItems ? parseFormConfig(data, formItems) : parseJsonConfig(data.config);
```

Comment:
Seems that whether to use form or JSON to configure the connector only depends on the existence of `formItems`; maybe we can use a toggle to switch between these two different ways of configuration?

Reply:
I think the existence of `formItems` is straitforword